staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -756,10 +756,7 @@ func (r *Request) watchInternal(ctx context.Context) (watch.Interface, runtime.D
 	isErrRetryableFunc := func(request *http.Request, err error) bool {
 		// The watch stream mechanism handles many common partial data errors, so closed
 		// connections can be retried in many cases.
-		if net.IsProbableEOF(err) || net.IsTimeout(err) {
```

Comment:
we had several discussions about this in the past, and there is no performance benefit since the code will be compiled anyway, and the majority of us find that  being explicit is easier to read and understand or that just is not meaningful to pollute the git history for a change like this

I suggest better find other places for contribute that need more help than changing this noop places

Reply:
no need to say sorry, I find this conversation interesting, thanks for the feedback